.github/workflows/linux.yml

```
@@ -37,6 +37,7 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
```

```suggestion
          # - "3.12"
```
Temporarily disable Python 3.12 for Linux CI